skyportal/handlers/api/telescope.py

```diff
@@ -135,23 +135,31 @@ def get(self, telescope_id=None):
         telescopes = []
         for telescope in data:
```

Comment:
Not sure if `telescope` will always have a `.to_dict()` method? 
```suggestion
        for telescope in data:
              if telescope is None:
                   continue
```